spec/features/group_spec.rb

```diff
@@ -0,0 +1,64 @@
+require 'rails_helper'
+
+# rubocop:disable Metrics/BlockLength
```

Comment:
- [ ] As mentioned by your previous code reviewer, don't try to bypass the `rubocop` linters to neglect the file check for the code quality. Please remove the comments from your all files, compromising the code quality is not a good idea because having lengthy code compromises the user `readability` or `code performance`. Kindly remove it.